api/src/decorators/get-user.decorator.ts

```diff
@@ -1,7 +1,7 @@
 import { createParamDecorator, ExecutionContext } from '@nestjs/common';
 import { User } from 'modules/users/user.entity';
 
-export const GetUser: ParameterDecorator = createParamDecorator(
+export const GetUser: any = createParamDecorator(
```

Comment:
I think it was me who put the initial type (ParamDecorator) which was not correct, since it returns void and requries 3 arguments...  
Probably this return type would be correct: ```const GetUser: (...dataOrPipes: unknown[]) => ParameterDecorator``